tests/deploy/test_gaie_deploy.py

```
@@ -0,0 +1,242 @@
+
+def _get_workspace_dir() -> str:
+    current = os.path.dirname(os.path.abspath(__file__))
+    while current != os.path.dirname(current):
+        if os.path.exists(os.path.join(current, "pyproject.toml")):
+            return current
+        current = os.path.dirname(current)
+    return os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+
+def _run_kubectl(*args, input_data=None):
+    cmd = ["kubectl"] + list(args)
```

Ruff (0.15.6)

[warning] Consider `["kubectl", *list(args)]` instead of concatenation

Replace with `["kubectl", *list(args)]`

(RUF005)